packages/components/organisms/f-content-cards/src/components/cardTemplates/StampCard1.vue

```diff
@@ -0,0 +1,315 @@
+                        ]"
+                    />
+                </div>
+            </div>
+        </div>
+    </component>
+</template>
+
+<script>
+import EmptyStamp from './images/stamp-empty-15.svg';
+import FullStamp from './images/stamp-full-15.svg';
+
+export default {
+    name: 'StampCard1',
+
+    components: {
+        EmptyStamp,
+        FullStamp
+    },
+
+    props: {
+        card: {
+            type: Object,
+            required: true
+        },
+
+        testId: {
+            type: String,
+            default: null
+        }
+    },
+
+    computed: {
+        title () {
+            return this.card.title;
+        },
+        description () {
+            return this.card.description;
+        },
+        image () {
+            return this.card.image;
+        },
+        url () {
+            return this.card.url;
+        },
+        discountPercentage () {
+            return this.card.discountPercentage;
+        },
+        earnedStamps () {
+            return this.card.earnedStamps;
+        },
+        expiryDate () {
+            return this.card.expiryDate;
+        },
+        expiryLine () {
+            return this.card.expiryLine;
+        },
+        isReadyToClaim () {
+            return !!this.card.isReadyToClaim;
+        },
+        totalRequiredStamps () {
+            // TBC by product - currently this.card.totalRequiredStamps is provided but we have no way of catering for
+            // any amount other than 5 here
+            return 5;
+        },
+        stamps () {
+            const stamps = [];
+
+            for (let i = 0; i < this.totalRequiredStamps; i++) {
+                const full = (i < this.earnedStamps);
+                stamps.push(full ? {
+                    stampImage: 'FullStamp',
+                    classSuffix: 'Full'
+                } : {
+                    stampImage: 'EmptyStamp',
+                    classSuffix: 'Empty'
+                });
+            }
+
+            return stamps;
+        },
+        statusText () {
+            return this.card.subtitle;
+        },
+        subStatusText () {
+            return this.description;
+        },
+
+        testIdForSection () {
+            return (section, index) => (this.testId
+                ? `${this.testId}--${section}${index === undefined ? '' : `--${index}`}`
+                : false);
+        }
+    },
+
+    mounted () {
+        this.onViewContentCard();
+    },
+
+    inject: [
+        'emitCardView',
+        'emitCardClick'
+    ],
+
+    methods: {
+        onViewContentCard () {
+            this.emitCardView(this.card);
+        },
+
+        onClickContentCard () {
+            this.emitCardClick(this.card);
+        }
+    }
+};
+</script>
+
+<style lang="scss" module>
+
+$stampCard-subStatus-colour: #017a39; /* $color-green in PIE - not in fozzie-colour-palette yet */
+$stampCard-expiryInfo-colour: $grey--dark;
+
+$stampCard-responsive-mobileViewBreakpoint: '<narrowMid';
```

Comment:
Why do we need a variable for `<narrowMid` here?

Reply:
A couple reasons - 

1. to avoid putting the 'magic' string everywhere we need it
2. to add some semantics around the meaning of this for the component - e.g. `mobileViewBreakpoint`. 
3. Essentially some insurance around easily changing this if UX decide the breakpoint is happening at the wrong screen width, as we've only been provided portrait and landscape designs.